api/BUILD

```diff
@@ -116,3 +116,15 @@ api_proto_library(
         ":discovery",
     ],
 )
+
+# TODO(htuch): Grow this to cover everything we want to generate docs for, so we can just invoke
+# bazel build //api --aspects tools/protodoc/protodoc.bzl%proto_doc_aspect  --output_groups=rst
```

Comment:
Will we be able to run this from the main repo? We will need to run it there so we can merge it in with the main doc build.

Reply:
Yeah, I think it will work with the usual symlink and repo threading hackery.